docker-compose-swarm.yml

```diff
@@ -0,0 +1,46 @@
+version: "3"
+services:
+  nest:
+    image: nest
+    deploy:
+      replicas: 1
+      restart_policy:
+        condition: on-failure
+    volumes:
+      - .env:/app/.env
+    depends_on:
+      - db
+
+  nginx:
```

Comment:
TODO: add volume for file uploads, and bind it to all services where needed in all docker-composes